src/features/Instructors/InstructorsDetailPage/index.scss

```diff
@@ -0,0 +1,66 @@
+@import "assets/variables.scss";
+@import "assets/colors.scss";
+
+.course-title {
+  max-width: 700px;
+}
+
+.card-container {
```

Comment:
If you are using the same styles for different pages, define them in the global styles so you can just use the class and not define them again for every page